src/main/java/org/killbill/billing/plugin/stripe/StripePaymentPluginApi.java

```diff
@@ -376,6 +352,44 @@ public void addPaymentMethod(final UUID kbAccountId, final UUID kbPaymentMethodI
         }
     }
 
+
+    private String createStripeCustomer(final UUID kbAccountId, final CallContext context, final RequestOptions requestOptions,
```

Comment:
For consistency in the order of arguments and style (see `executeFollowUpTransaction`):

```
private String createStripeCustomer(final UUID kbAccountId,
                                    final String paymentMethodIdInStripe,
                                    final String defaultStripeId,
                                    final RequestOptions requestOptions,
                                    final Iterable<PluginProperty> allProperties
                                    final CallContext context) throws StripeException, PaymentPluginApiException {
```